### ISSUE TITLE
Question: Will Word outputs support comments?

### ISSUE THREAD

**Issue by spgarbet** [davidgohel/officer#52]
I'm working on producing tables in Word from R. A big problem we have is tracing statistical report content that is then copy and pasted into other Word documents. I've got a method to generate unique keys for statistical content, and trace it with comments. Then later if data is changed, we can automatically substitute the content back in the final document.

My ideal Word generation from R needs are
  * Has no crazy run time compile requirements for libraries that I can't get to compile easily.
  * Can create tables
  * Can comment content inside tables
  * Can generate Word from pandoc cleanly
  * Full control of table styles in document

Looks like you've hit the first two for me. The 3rd is really the big need. The last 2 I can live without and work toward later.

TL;DR will the library eventually support commenting?

**Comment by davidgohel** [comment]
Good idea but unfortunately, there is no plan for now to integrate comments in Word documents.
